.github/workflows/kvrocks.yaml

```diff
@@ -327,7 +327,20 @@ jobs:
         with:
           name: sonarcloud-data
           path: ${{ env.SONARCLOUD_OUTPUT_DIR }}
-
+          
+      - name: Save PR info
+        if: ${{ (github.event_name == 'pull_request') && matrix.sonarcloud }}
+        run: |
+          mkdir -p ./pr
+          echo "number=${{ github.event.number }}" >> ./pr/info
+          
+      - name: Upload PR info
+        uses: actions/upload-artifact@v3
+        if: ${{ matrix.sonarcloud }}
+        with:
+          name: pr
+          path: pr/
```

Comment:
You can just add a new file to ${{ env.SONARCLOUD_OUTPUT_DIR }}.

Reply:
You are right, Thank you.